lib/mix/lib/mix/tasks/local.hex.ex

```diff
@@ -16,14 +16,21 @@ defmodule Mix.Tasks.Local.Hex do
 
     * `--force` - forces installation without a shell prompt; primarily
       intended for automation in build systems like `make`
+
+  ## Mirrors
+
+  If you want to define a new mirror for Hex please set the `HEX_CDN`
+  environment variable.
```

Comment:
I think this wording can be improved, we are not defining a new mirror :). What about: "If you want to change the default mirror (http://s3.amazonaws.com/s3.hex.pm) to use for fetching Hex please set the `HEX_CDN` environment variable."?